ui/src/environments/environment.ts

```diff
@@ -11,4 +11,5 @@ export const environment = {
   scope: '',
   additionalColumns: null,
   entryPoint: 'jobs',
+  serverUrl: ''
```

Comment:
You need to add this default value to both dsub environment files as well, right now running the google and local servers fails. Also trailing comma please.